### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by brentwpeterson**
## This PR is a:

- [x] New topic
- [x] Content update
- [x] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will update the community page to allow for best practices section

## Additional information

List all affected URLs 

- ...
- ...

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/devdocs?pullRequest=3111) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/devdocs?pullRequest=3111) before we can accept your contribution.<br/>

**Comment by magento-engcom-team** [comment]
@brentwpeterson thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

File: community/resources/index.md [hunk 1]
```diff
@@ -21,6 +21,13 @@ Does it fill gaps in our current docs and provide important information?
 **Note:** Magento does not officially endorse any of the blogs, books, presentation, or podcasts linked below.
 They are listed here because the content has been well received within the Magento community.
 </div>
```
Comment (lorikrell) [inline community/resources/index.md]: Can you add a blank line here? Between the </div> and new heading? It'll just append the text when generated without putting it into that format.

Comment (brentwpeterson) [reply community/resources/index.md]: I have added the line

**Comment by brentwpeterson** [comment]
Thanks, I signed the CLA and I added the blank line. I would also like to add navigation to already existing best practices in DevDocs

**Comment by lorikrell** [comment]
I'll get a TOC and navigation all fixed in a new PR. I'll add you and others interested for reviewing, etc. Thanks!

**Comment by dshevtsov** [comment]
@lorikrell please add `whatsnew`

**Comment by dshevtsov** [comment]
The PR doesn't seem to be Major update. It doesn't introduce any new information.
I would mark it as **Editorial**.

**Comment by lorikrell** [comment]
The whatsnew for this is covered with https://github.com/magento/devdocs/pull/3121 thanks!

**Pull request merged**